src/store/selectors.ts

```diff
@@ -213,6 +215,7 @@ export const selectPlayerDisplayData = createSelector(
         cards: player.hand,
         isCurrentPlayer,
         isFirstPersonTeammate:
+          game.isTeammateRevealed &&
```

Comment:
`isFirstPersonTeammate` is checking if given player is your teammate. What you are looking for is `isTeammate` (see tils.gameUassignTeamsByTeammateCard())